addons/pos_loyalty/static/tests/tours/pos_loyalty_reward_button_tour.js

```diff
@@ -80,11 +80,10 @@ registry.category("web_tour.tours").add("PosLoyaltyFreeProductTour", {
             ProductScreen.clickDisplayedProduct("Magnetic Board"),
             ProductScreen.selectedOrderlineHas("Magnetic Board", "2.00"),
             ProductScreen.clickDisplayedProduct("Magnetic Board"),
-            ProductScreen.selectedOrderlineHas("Magnetic Board", "3.00"),
```

Comment:
This is expected after the patch because the selected line would be the line added to balance the reward line